martin/src/file_config.rs

```diff
@@ -264,7 +270,11 @@ async fn resolve_int<T: SourceConfigExtras>(
                 let dup = if dup { "duplicate " } else { "" };
                 let id = idr.resolve(&id, url.to_string());
                 configs.insert(id.clone(), source);
-                results.push(cfg.custom.new_sources_url(id.clone(), url.clone()).await?);
+                match cfg.custom.new_sources_url(id.clone(), url.clone()).await {
```

Comment:
i'm not a big fan of all this dup code for each call to `new_sources` - i wonder if we can do this logic elsewhere, and here keep the simple `?` operator as before?